package.json

```diff
@@ -31,36 +31,36 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@babel/core": "^7.6.3",
+    "@babel/core": "^7.6.0",
     "@babel/polyfill": "^7.6.0",
     "@babel/preset-env": "^7.6.0",
     "@types/custom-functions-runtime": "^1.5.0",
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.7.5",
-    "@types/office-js": "^1.0.40",
+    "@types/office-js": "^1.0.37",
     "@types/office-runtime": "^1.0.8",
     "babel-loader": "^8.0.6",
     "clean-webpack-plugin": "^2.0.1",
-    "copy-webpack-plugin": "^5.0.4",
+    "copy-webpack-plugin": "^5.1.1",
     "custom-functions-metadata-plugin": "^1.0.20",
     "eslint-config-office-addins": "^1.0.11",
     "file-loader": "^4.2.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
     "mocha": "^6.2.0",
-    "office-addin-cli": "^1.0.4",
-    "office-addin-debugging": "^3.0.18",
-    "office-addin-dev-certs": "^1.4.11",
-    "office-addin-lint": "^1.0.15",
-    "office-addin-prettier-config": "^1.0.11",
-    "office-addin-test-helpers": "^1.0.14",
-    "office-addin-test-server": "^1.0.17",
+    "office-addin-cli": "^1.0.7",
+    "office-addin-debugging": "^3.0.23",
+    "office-addin-dev-certs": "^1.4.14",
+    "office-addin-lint": "^1.0.19",
+    "office-addin-prettier-config": "^1.0.12",
+    "office-addin-test-helpers": "^1.0.17",
+    "office-addin-test-server": "^1.0.21",
     "source-map-loader": "^0.2.4",
-    "ts-loader": "^6.2.0",
+    "ts-loader": "^6.1.0",
     "ts-node": "^8.4.1",
-    "typescript": "^3.6.4",
-    "webpack": "^4.41.0",
-    "webpack-cli": "^3.3.9",
+    "typescript": "^3.6.3",
```

Comment:
why is Typescript going backwards?

Reply:
Mistake when resolving merge conflicts -- good catch. will fix and push change